src/main/java/reposense/authorship/FileInfoAnalyzer.java

```diff
@@ -100,7 +101,7 @@ private static void aggregateBlameAuthorInfo(RepoConfiguration config, FileInfo
      * Returns the analysis result from running git blame on {@code filePath}.
      */
     private static String getGitBlameResult(RepoConfiguration config, String filePath) {
-        return CommandRunner.blameRaw(config.getRepoRoot(), filePath);
+        return blame(config.getRepoRoot(), filePath);
```

Comment:
Unintentionally use of static import? 
`import static reposense.git.GitBlame.blame;`

Only `GitBlame` is using it.